bin/install-services

```diff
@@ -8,7 +8,15 @@ grep 'name:' config/services.js | \
 	do
 		pushd $service
 		echo "Installing service $service"
-		npm install --quiet
+		case $service in
+			web)
+				# install webpack and friends from dev-dependencies.
+				npm ci
```

Comment:
Should we keep using `--quiet` for manageable logs?

Reply:
I tried `npm ci --quiet` locally for the notifications service and it did not make any difference :shrug:

Reply:
Weird, according to npm v7 `--quiet` is still equivalent to --loglevel warn` https://docs.npmjs.com/cli/v7/using-npm/config